libs/perun/components/src/lib/entity-search-select/entity-search-select.component.ts

```diff
@@ -58,10 +62,6 @@ export class EntitySearchSelectComponent<T extends PerunBean>
   ) => entity.status;
 
   ngOnInit(): void {
-    this.entitiesCtrl.valueChanges
-      .pipe(distinctUntilChanged())
-      .subscribe((entity: T) => this.entitySelected.emit(entity));
```

Comment:
This code prevents emitting of the same value multiple times. This is quite important and can cause errors in some places. If you insist on removing it, make sure you handle it the onChange method.